security/pkg/nodeagent/sds/sdsservice.go

```diff
@@ -674,7 +674,7 @@ func sdsDiscoveryResponse(s *model.SecretItem, resourceName, typeURL string) (*d
 	return resp, nil
 }
 
-func newSDSConnection(stream v2.DiscoveryStream) *sdsConnection {
+func newSDSConnection(stream xds.DiscoveryStream) *sdsConnection {
```

Comment:
Here the PR changes "v2.DiscoveryStream" to "xds.DiscoveryStream". DiscoveryStream is specific to "istio.io/istio/pilot/pkg/proxy/envoy/v2" but "xds.DiscoveryStream" does not contain the version. Can you explain how the version is represented in "xds.DiscoveryStream"?

Reply:
Same explanation as above.